docs/discover/document-explorer.asciidoc

```diff
@@ -99,26 +94,43 @@ the documents that occurred before and after it.
 . Click the expand icon
 image:images/expand-icon-2.png[double arrow icon to open a flyout with the document details].
 +
+You can view the document in two ways. The **Table** view displays the document fields row-by-row.
```

Comment:
It might be worth to mention that the flyout offers pagination for the current list of records, which meant you can easily navigate to next previous or next documents in the list using the arrow navigation elements